### ISSUE TITLE
try to detect forgotten commands 

### ISSUE THREAD

**Issue by sknorr** [koalaman/shellcheck#1637]
#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/bin/bash
"this is a random message"

```

#### Here's what shellcheck currently says:

Nothing.

#### Here's what I wanted or expected to see:

Since my script is just a quoted string that also includes spaces, I'd like shellcheck to give me a warning like: 
"W: Did you forget to use the `echo` command?"
